### DISCUSSION TITLE
indent/group function

### DISCUSSION THREAD

**Discussion by Elementari** [mbnuqw/sidebery#907] [Ideas]
v4.10.2

(just some feedback from the average user)

Hello,
I don't understand why in the context menu of a tab, there is no menu item to 
a) unindent all tabs within a tab hierarchie, e.g.

TAB A
--SUBTAB B
--SUBTAB C
TAB D

-> right click tab1,  then select "dissolve hierarchy" (or sth. like that), causing this result:

TAB A
TAB B
TAB C
TAB D

-------
b) and:
rightlick SUBTAB B, then "remove from hierachy" causing:

TAB A
--SUBTAB C
TAB B
TAB D

------

Even if this this possible already, it's difficult to get done or not obvious to average users like myself (except if I'm below average haha)

**Comment by notDavid** [comment]
For anyone stumbling upon this, note that it's possible with a keyboard shortcut:

`Settings > Keybindings > Moving tabs > Increase tab's tree level (indent)`
`Settings > Keybindings > Moving tabs > Decrease tab's tree level (indent)`

(while using this keybinding, you can also select multiple tabs with the `Shift` key, obviously.)
